dashboard/views.py

```diff
@@ -64,46 +64,30 @@ def gpa_map(grade):
         return 'C'
     elif grade_float < 90:
         return 'B'
-    else:
+    elif grade_float > 90:
         return 'A'
+    else:
+        return NO_GRADE_STRING
+
+def get_current_week_number(request):
+    # get term start date
+    # TODO: term id hardcoded now
```

Comment:
Should probably create these TODO's as new issues so we don't forget about them.

Reply:
now added a config variable "CURRENT_CANVAS_TERM_ID" into .env.sample, and the tool now read the setting from the .env file.